Week_7.java

```diff
@@ -0,0 +1,166 @@
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+
+/*
+
+Microsoft Interview Coaching Pilot 2018 (MICP 2018)
+Author: Aishna Gupta
+Language: JAVA
+
+************WEEK 6************
+
+Problem description: Design a vending machine.
+Clarify: No more restrictions, just design a general vending machine we are familiar with.
+*/
+
+public class Week_7 {
+	
+	
+	public class Item{
+		private String Item_Id;
+		private String Item_Name;
+		private int Item_Quantity;
+		private int Item_Price;
+		
+		Item(String id, String name, int quantity,int price){
+			Item_Id = id;
+			Item_Name = name;
+			Item_Quantity = quantity;
+			Item_Price = price;
+		}
+		
+		public int getPrice(){
+			return Item_Price;
+		}
+		
+		public void setPrice(int price){
+			 Item_Price = price;
+		}
+		
+		public int getQuantity(){
+			return Item_Quantity;
+		}
+		
+		public void setQuantity(int quantity){
+			 Item_Quantity = quantity;
+		}
+		
+		public String getName(){
+			return Item_Name;
+		}
+		
+		public void setName(String name){
+			 Item_Name = name;
+		}
+
+		public String getId(){
+			return Item_Id;
+		}
+		
+		public void setId(String Id){
+			 Item_Id = Id;
+		}
+		
+		
+	}
+	
+	public class Change{
+		Item item;
+		int change;
+	}
+	
+	public enum CASH{
```

Comment:
enum type names shouldn't be in all caps, but the values can be:
https://stackoverflow.com/questions/3069743/coding-conventions-naming-enums?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa